magda-web-client/src/helpers/CsvDataLoader.ts

```diff
@@ -192,6 +206,10 @@ class CsvDataLoader {
                 }).bind(this),
                 complete: (() => {
                     try {
+                        if (this.skipComplete) {
+                            this.skipComplete = true;
```

Comment:
Why set it to true if it's already true?

Reply:
Good pick-up, it should be `this.skipComplete = true;`
It still works as it's been reset in `resetDownloadData` as well (i.e. I was lucky 😄 ). 
I will correct this~